internal/db/migrations/004_pubkeys.sql

```diff
@@ -17,7 +17,8 @@ CREATE TABLE pubkey_resources
   pubkey_id BIGINT NOT NULL REFERENCES pubkeys(id),
   provider INTEGER NOT NULL CHECK (valid_provider(provider)),
   source_id TEXT NOT NULL,
-  handle TEXT NOT NULL
+  handle TEXT NOT NULL,
+  region TEXT NOT NULL
```

Comment:
I'm so sorry you've had to be the first, but we'll need to do a proper migration now that we are in stage :(

Reply:
To elaborate, create a new SQL file with a new sequence numer and then do ALTER TABLE command to add a new column.